src/utils/sformat.fs

```diff
@@ -1194,7 +1194,7 @@ namespace Microsoft.FSharp.Text.StructuredFormat
             | :? bool   as b -> (if b then "true" else "false")
             | :? char   as c -> "\'" + formatChar true c + "\'"
             | _ -> try  let text = obj.ToString()
-                        text
+                        if text = null then "" else text
```

Comment:
I actually checked that, and, at least for string, this comparison just compiles to `string.Equals(text, null)` which should be optimal.